app/views/plans/index.html.erb

```diff
@@ -4,10 +4,9 @@
     <!-- if the user has projects -->
     <p>
       <% if @plans.count > 0 %>

```

Comment:
not sure when this got merged but we should get the habit of using length instead of count to avoid an extra query